### ISSUE TITLE
Change average response time to mean

### ISSUE THREAD

**Issue by kmcurry** [kmcurry/property-buddy#32]
At national Day of civic hacking 2017 a city of Virginia Beach is data scientist told me that average response time is a bad indicator due to situational factors. For example, in an emergency situation and officer might have to choose between logging the arrival time and responding. Mean should be used instead as a better indicator to throw out outliers

**Comment by PhoenixFireVB** [comment]
I'm assuming he meant the trimmed mean, or some form of standard deviation.  Can you describe the formula used vs. what was proposed?

**Comment by kmcurry** [comment]
I'm just summing the values and dividing by number of values. Natasha Singh-Miller suggested a different method but I don't know what. We can ask her. We should filter outliers for sure.

**Comment by kmcurry** [comment]
I've just realized I've been using the wrong terms here. Current method is sum of values divided by number of values (average, mean). We need a better method that discards outliers such as extremely long response times and possibly instantaneous (0) response times.

**Comment by PhoenixFireVB** [comment]
Yeah that's what I thought you meant.  I wasn't sure if you had a specific method in mind, or just something better than average.
